src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -157,6 +157,10 @@
     }
 }
 
+.bcs-is-wider .bcs-activity-feed {
```

Comment:
this was the only tab you had to set the width ?

Reply:
Is removing the override altogether an option?